### PR TITLE
Typo fixes.

### PR DESCRIPTION
**Pull request by eforgacs**

### CODE REVIEW

**Comment by eforgacs** [comment]
I actually have a few more typo fixes than just these; would you like me to add them to this PR, and make one big "typo fix" PR? Up to you.

**Comment by ppizarror** [comment]
> @ppizarror I actually have a few more typo fixes than just these; would you like me to add them to this PR, and make one big "typo fix" PR? Up to you.

Let's use this PR. thanks!

**Comment by eforgacs** [comment]
Okay, this should have gotten a big chunk of the typos out of the way.

File: pygame_menu/menu.py [hunk 1]
```diff
@@ -58,7 +58,7 @@ class Menu(object):
     :type width: int, float
     :param title: Title of the Menu (main title)
     :type title: str
-    :parama auto_center_v: Auto centers the menu on the vertical position after a widget is added/deleted
```
Comment (eforgacs) [inline pygame_menu/menu.py:61]: I noticed that when I changed this from "parama" to "param", it didn't pick up on this parameter. Has it been removed?

Comment (ppizarror) [reply pygame_menu/menu.py:61]: Yes this has been removed,

Comment (ppizarror) [reply pygame_menu/menu.py:61]: This should be `center_content`

Comment (eforgacs) [reply pygame_menu/menu.py:61]: Okay. I'll change it to match.

**Pull request merged**